### DISCUSSION TITLE
For the onboarding repo is it possible to default to `.renovaterc.json`?

### DISCUSSION THREAD

**Discussion by paddyroddy** [renovatebot/renovate#25376] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

I would like the onboarding PR to be in a `renovaterc.json` file, rather than `renovate.json` as this is my desired format.

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
It's this setting, which is currently global-only: https://docs.renovatebot.com/self-hosted-configuration/#onboardingconfigfilename

Please watch #22212 as this would be the type of setting you could control yourself once it's done. Fro now it's not possible to change through config

**Reply by paddyroddy** [reply]
Okay, thank you. So it's a self-hosted option only.

**Reply by rarkins** [reply]
Until that feature lands
